src/components/number-picker/number-picker.component.ts

```diff
@@ -127,7 +127,10 @@ export class NumberPickerComponent implements ControlValueAccessor, OnDestroy, O
         }
 
         if (!this.disabled) {
-            this.value = Math.max(Math.min(this.value + this.step, this.max), this.min);
+            this.value = Math.max(
```

Comment:
I think you should maybe increase the line length of your code formatter, its seems a bit over keen to wrap lines. Lines this short should probably remain all on one line to make them easier to read.